webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -820,6 +820,12 @@ export default class SuggestionBox extends React.PureComponent {
                     ref={this.inputRef}
                     autoComplete='off'
                     {...props}
+                    aria-owns='suggestionList'
```

Comment:
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/ has a note recommending `aria-controls` for this instead of `aria-owns`:

> In ARIA 1.0, the combobox referenced its popup with [aria-owns](https://w3c.github.io/aria/#aria-owns) instead of [aria-controls](https://w3c.github.io/aria/#aria-controls). While user agents might support comboboxes with [aria-owns](https://w3c.github.io/aria/#aria-owns) for backwards compatibility with legacy content, it is strongly recommended that authors use [aria-controls](https://w3c.github.io/aria/#aria-controls).

Would you mind changing that?